docs/server-side/v2/policy/create-template-via-api.mdx

```diff
@@ -595,24 +595,24 @@ curl --location --request POST 'https://api.100ms.live/v2/templates' \
-| name                    | `string`  | The name you can assign to identify the HLS destination                                                                    | No       |
-| maxDuration             | `int`     | Sets the maximum duration, in seconds, of the recording. For example, 1800 seconds(30 mins)                                | No       |
-| layers                  | `object`  | Object of type `layers`. This can be used to configure the various dimensions to be supported for HLS streaming.           | No       |
-| playlistType            | `string`  | Indicates the type of playlist. Can be of type `event` or `live`.                                                          | No       |
-| numPlaylistSegments     | `int`     | Indicates the number of media segments in the playlist. Applicable only for `live` playlistType. Min/max/default = 3/12/5. | No       |
-| videoFrameRate          | `int`     | Set this argument to configure the frames per second (or FPS) for your stream. For example, 25                             | No       |
-| enableMetadataInsertion | `boolean` | Set this to true to enable metadata insertion along with the stream.                                                       | No       |
-| enableStaticUrl         | `boolean` | If recording is required this can be set as true. This value has no effect on streaming.                                   | No       |
-| recording               | `object`  | Object of type `recording`. This can be used to enable HLS recording and configure the layers, thumbnails, etc.            | No       |
+| Name                    | Type      | Description                                                                                                                                                                                        | Required |
+| :---------------------- | :-------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| name                    | `string`  | The name you can assign to identify the HLS destination                                                                                                                                            | No       |
+| maxDuration             | `int`     | Sets the maximum duration, in seconds, of the recording. For example, 1800 seconds(30 minutes)                                                                                                     | No       |
+| layers                  | `object`  | Object of type `layers`. This can be used to configure the various dimensions to be supported for HLS streaming. The layers are expected to be in descending order of resolution and video bitrate | No       |
```

Comment:
aren't we reordering the layers on the backend?

Reply:
No